src/superphot_plus/import_ztf_from_alerce.py

```diff
@@ -2,18 +2,22 @@
 data from the Alerce API."""
 
 import csv
-import glob
 import os
 import zipfile
 
 import numpy as np
 from alerce.core import Alerce
 from antares_client.search import get_by_ztf_object_id
 
+from superphot_plus.file_paths import *  # pylint: disable=wildcard-import
 from superphot_plus.utils import convert_mags_to_flux, get_band_extinctions
 
 alerce = Alerce()
 MIN_PER_FILTER = 5
+EXTRACT_DIR = ""
```

Comment:
move these to file_paths, remove redundant filepaths if possible

Reply:
Thanks, Kaylee! Considering these path values are unknown I decided to pass them as function parameters for the time being. Is that okay? Hopefully this will ease testing the methods that'll make use of these paths!